packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -444,16 +444,12 @@ mixin WidgetsBinding on BindingBase, ServicesBinding, SchedulerBinding, GestureB
         name: 'fastReassemble',
         callback: (Map<String, Object> params) async {
           final String? className = params['className'] as String?;
-          void markElementsDirty(Element element) {
-            if (element.widget.runtimeType.toString() == className) {
-              element.markNeedsBuild();
-            }
-            element.visitChildElements(markElementsDirty);
+          BindingBase.debugReassembleConfig = DebugReassembleConfig(widgetName: className);
+          try {
+            await reassembleApplication();
+          } finally {
+            BindingBase.debugReassembleConfig = null;
           }
-          if (renderViewElement != null) {
-            markElementsDirty(renderViewElement!);
-          }
-          await endOfFrame;
```

Comment:
Did you mean to remove this `await`?

Reply:
Yes, this method is intended to mirror https://github.com/flutter/flutter/blob/85df73d0e34e1a80d3e468303926434deb790ef9/packages/flutter/lib/src/foundation/binding.dart#L165-L169 but with an extra config

Will add a comment

Reply:
comment added